From 545e646a80fed3dd353ac31d83b2c04f202f1bf5 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Mon, 2 Oct 2017 15:19:43 -0400
Subject: [PATCH] WRS:
 0002-Add-nfs-utils-update-nfsmount.conf-with-required-con.patch

---
 SPECS/nfs-utils.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/nfs-utils.spec b/SPECS/nfs-utils.spec
index 282511e..7793ee1 100644
--- a/SPECS/nfs-utils.spec
+++ b/SPECS/nfs-utils.spec
@@ -152,6 +152,7 @@ Patch1003: nfs-utils-1.2.5-idmap-errmsg.patch
 # Titanium Cloud Patches
 Patch2000: nfs-utils-support-mountprog-and-nfsprog-options.patch
 Patch2001: nfs-utils-do-not-pass-CFLAGS-to-native.patch
+Patch2002: nfs-utils-update-nfsmount.conf-with-required-config.patch
 
 Group: System Environment/Daemons
 Provides: exportfs    = %{epoch}:%{version}-%{release}
@@ -430,6 +431,7 @@ This package also contains the mount.nfs and umount.nfs program.
 # Titanium Cloud support mountprog and nfsprog options
 %patch2001 -p1
 # Titanium Cloud do not pass CFLAGS to native 
+%patch2002 -p1
 
 # Remove .orig files
 find . -name "*.orig" | xargs rm -f
-- 
2.7.4

